### PR TITLE
Use different table names to prevent a race when running in parallel

### PR DESCRIPTION
**Pull request by kabunchi**

### CODE REVIEW

File: presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java [hunk 1]
```diff
@@ -7442,13 +7442,14 @@ private void testTimestampPrecisionInsert(Session session, HiveStorageFormat sto
             return;
         }
 
-        String createTable = "CREATE TABLE test_timestamp_precision (ts TIMESTAMP) WITH (format = '%s')";
-        @Language("SQL") String insert = "INSERT INTO test_timestamp_precision VALUES (TIMESTAMP '%s')";
+        String createTable = "CREATE TABLE test_timestamp_precision_insert (ts TIMESTAMP) WITH (format = '%s')";
```
Comment (kokosing) [inline presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java:7445]: We typically are using `io.prestosql.testing.sql.TestTable#randomTableSuffix`

File: presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java [hunk 2]
```diff
@@ -7467,20 +7468,21 @@ private void testTimestampPrecisionCtas(Session session, HiveStorageFormat stora
             return;
         }
 
-        String createTableAs = "CREATE TABLE test_timestamp_precision WITH (format = '%s') AS SELECT TIMESTAMP '%s' ts";
+        String createTableAs = "CREATE TABLE test_timestamp_precision_ctas WITH (format = '%s') AS SELECT TIMESTAMP '%s' ts";
 
         testTimestampPrecisionWrites(
                 session,
+                "test_timestamp_precision_ctas",
```
Comment (kokosing) [inline presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java:7475]: please extract table name variable

File: presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java [hunk 3]
```diff
@@ -7442,13 +7442,14 @@ private void testTimestampPrecisionInsert(Session session, HiveStorageFormat sto
...
         }
 
-        String createTable = "CREATE TABLE test_timestamp_precision (ts TIMESTAMP) WITH (format = '%s')";
-        @Language("SQL") String insert = "INSERT INTO test_timestamp_precision VALUES (TIMESTAMP '%s')";
+        String createTable = "CREATE TABLE test_timestamp_precision_insert (ts TIMESTAMP) WITH (format = '%s')";
+        @Language("SQL") String insert = "INSERT INTO test_timestamp_precision_insert VALUES (TIMESTAMP '%s')";
 
         testTimestampPrecisionWrites(
                 session,
+                "test_timestamp_precision_insert",
```
Comment (kokosing) [inline presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java:7450]: please extract table name variable

File: presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java (hunk 1 above)
Comment (kabunchi) [reply presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java]: fixed and pushed

**Review by kokosing — APPROVED** [review]

**Comment by kokosing** [comment]
Merged, thanks!

**Pull request merged**